site/content/docs/cli/command-line-reference.md

```diff
@@ -0,0 +1,90 @@
+---
+title: Command Line Reference
+weight: 99
+menu:
+  docs:
+    parent: "CLI"
+---
+
+Dry run all checks in your repo:
+
+```bash
+npx checkly test
+```
+
+Dry run checks that have `product` and `api` in the file name
+
+```bash
+npx checkly test product api
+```
+
+Dry run checks while passing a specific URL and a password. These variables are available on `process.env`
+
+```bash
+npx checkly test --env "ENVIRONMENT_URL=https://preview.acme.com" --env PASSWORD=doremiabc123
+```
+
+Dry run all checks against a specific location:
+
+```bash
+npx checkly test --location eu-west-1
+```
+
+Preview the differences between your actual configuration and your account
```

Comment:
I was confused about what this meant and had to run it. A sentence explaining it might be helpful.

<img width="732" alt="image" src="https://user-images.githubusercontent.com/962099/218426104-d6f62247-11b9-4644-9e09-20f946ae734f.png">